### ISSUE TITLE
cant login getting error

### ISSUE THREAD

**Issue by crazyRM** [UnamSanctam/UnamWebPanel#193]
hey im getting a error when trying to login and yes password is correct using all default stuff from this repo

"SCRIPT5009: 'unam_jsonAjax' is undefined
login.php (78,5)"

**Comment by UnamSanctam** [comment]
What browser are you using? And also make sure that all the files are uploaded.

**Comment by crazyRM** [comment]
Opera GX and yes all the files are uploaded

**Comment by UnamSanctam** [comment]
Looks like you can't load the unam_lib.js, try another browser if you are sure all the files are there.

**Comment by crazyRM** [comment]
just tried microsoft edge and its not working

**Comment by UnamSanctam** [comment]
Then you have done something wrong with the files.

**Comment by crazyRM** [comment]
I am using all of the files from your newest release From RileyOn 14 Sep 2022, at 4:56 pm, Unam Sanctam ***@***.***> wrote:﻿
Then you have done something wrong with the files.

**Comment by UnamSanctam** [comment]
Then you'll have to post your website for me to see, there's something wrong about the setup (since the files themselves obviously work).

**Comment by crazyRM** [comment]
its on xampp would you be able to continue this convo on discord? as i dont wanna post my servers info here

**Comment by UnamSanctam** [comment]
And you extracted the UnamWebPanel.zip https://github.com/UnamSanctam/UnamWebPanel/releases into the htdocs folder? I don't have a Discord anymore since I got banned for helping people with the miner.

**Comment by crazyRM** [comment]
yep put it in htdocs i get the login screen put in the password and it fails in console no error log no wrong password message
nothing just nothing very weird
